collector/src/cgroups.h

```diff
@@ -53,6 +57,133 @@ std::string CGroupDetected2string(CGroupDetected k);
 // Types
 //------------------------------------------------------------------------------
 
+#ifdef PROMETHEUS_SUPPORT
+/* structure for prometheus output : CPU utilization as reported by cpuacct cgroup */
+static const prometheus_kpi_descriptor g_prometheus_kpi_cgroup_cpu[] = {
+    // cgroup : cpu
+    { "cgroup_cpuacct_stats_user", prometheus::MetricType::Gauge, "CPU time consumed by tasks in user mode" },
```

Comment:
metrics should have a clear measurement units; for example it's not clear if 'cgroup_cpuacct_stats_user' is a percentage, time in msec, time in secs, etc.. following https://prometheus.io/docs/practices/naming/ we can append e.g. _sec to indicate this is measured in secs

Reply:
I don't remember though if we said in the past that's it better to keep 100% consistency with the JSON structure... is that the case?